File: tests/fixtures/typescript_simple/src/services/email.ts
import { UserType } from "../types/user";

export function sendWelcomeEmail(user: UserType): boolean {
  return sendEmail(user.email, "Welcome!", `Hello ${user.name}`);
}

function sendEmail(to: string, subject: string, body: string): boolean {
  return to.length > 0 && subject.length > 0 && body.length > 0;
}

File: tests/fixtures/python_simple/services/email.py
from models.user import User

def send_welcome_email(user: User) -> bool:
    """Send a welcome email to a new user."""
    return send_email(user.email, "Welcome!", f"Hello {user.name}")

def send_email(to: str, subject: str, body: str) -> bool:
    """Send an email via SMTP."""
    return True
